[ref #87] test/integration/member_hoge を書く

yegawa opened the pull request:
`member_register_test.rb` 書いた。
色々抜けてたりしてやばかった

yegawa committed:
[ref #87] test/integration/member_hoge を書く

`member_register_test.rb` 書いた。
色々抜けてたりしてやばかった

okabi commented on app/controllers/concerns/accounts_common.rb:
```diff
@@ -52,5 +52,13 @@ def failed_password_reset
       render 'new'
     end
+
+    def redirect_to_home_page_after_logged_in(user_classification)
```
このメソッド名だとログイン処理も含んでいるように見える。
`user_classification` 渡してる時点でそれに対するホームだという意味は取れるので、
`redirect_to_home` で十分だと思います。
非ログイン状態で呼ばれたくないメソッドなら、メソッド内でログイン状態を確認して処理分岐するのが良いかなぁ…。

okabi reviewed (commented):
小言みたいなレビューです。。。

yegawa committed:
[ref #87] test/integration/member_hoge を書く

redirect_to_home_page_after_logged_in => redirect_to_home
に変更して、ログインチェックも行うようにした。

okabi reviewed (approved):
良さそう 👍
